frontend/apps/web/app/(mgmt)/[account]/connections/[id]/page.tsx

```diff
@@ -69,16 +69,17 @@ export default function ConnectionPage({ params }: PageProps) {
       </div>
     ),
   });
+
+  const basePath = `/${account?.name}/connections/${data?.connection?.id}`;
```

Comment:
what's this?

Reply:
i was starting to build out the permissions on a new tab and then decided to move it, left over code from that.